File: 프로그래머스/unrated/181926. 수 조작하기 1/수 조작하기 1.js
function solution(n, control) {
    control.split("").map( (item) => {
      if (item === 'w') n++;
      if (item === 's') n--;
      if (item === 'd') n = n + 10;
      if (item === 'a') n = n - 10;
   })
   return n;
}

File: 프로그래머스/unrated/181926. 수 조작하기 1/수 조작하기 1.java
class Solution {
    public int solution(int n, String control) {
        String[] items = control.split("");
        for (String item : items) {
            switch (item) {
                case "w":
                    n++;
                    break;
                case "s":
                    n--;
                    break;
                case "d":
                    n = n + 10;
                    break;
                case "a":
                    n = n - 10;
                    break;
            }
        }
        return n;
    }
}